tool/travis.sh

```diff
@@ -1,15 +1,109 @@
 #!/bin/bash
-
-# Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
-# for details. All rights reserved. Use of this source code is governed by a
-# BSD-style license that can be found in the LICENSE file.
+# Created with https://github.com/dart-lang/mono_repo
 
 # Fast fail the script on failures.
 set -e
 
-# Use only one worker per task to limit resource usage on travis
-export BUILD_MAX_WORKERS_PER_TASK=1
+if [ -z "$PKG" ]; then
+  echo -e "\033[31mPKG environment variable must be set!\033[0m"
+  exit 1
+elif [ -z "$TASK" ]; then
+  echo -e "\033[31mTASK environment variable must be set!\033[0m"
+  exit 1
+fi
 
 pushd $PKG
 pub upgrade
-./tool/travis.sh
+
+case $PKG in
+_goldens) echo
+  echo -e "\033[1mPKG: _goldens\033[22m"
+  echo -e "  Running `tool/travis.sh`"
+  tool/travis.sh
+  ;;
+_tests) echo
+  echo -e "\033[1mPKG: _tests\033[22m"
+  echo -e "  Running `tool/travis.sh`"
+  tool/travis.sh
+  ;;
+angular_test) echo
+  echo -e "\033[1mPKG: angular_test\033[22m"
+  echo -e "  Running `tool/travis.sh`"
+  tool/travis.sh
+  ;;
+angular_router) echo
+  echo -e "\033[1mPKG: angular_router\033[22m"
+  echo -e "  Running `tool/travis.sh`"
```

Comment:
the script in the `ticks` is evaluated – I'm a moron

https://github.com/dart-lang/mono_repo/pull/49